### PR TITLE
US-24 Entity-Tag - Create new record and save it.

### PR DESCRIPTION
**Pull request by LMalushkina**
US-24 Entity-Tag - Create new record and save it, https://trello.com/c/CZH0ItMe/434-at241javascript-entity-tag-create-record.

### CODE REVIEW

**Review by MariaMoroz — APPROVED** [review]

File: test/specs/entityTagNewRecord.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        $(selectors.entityTag.date).click;
+    });
+    it('06/LP-Should create new DateTime', () => {
+        $(selectors.entityTag.datetime).click;
+    });
+    it('07/LP-Should choose user', () => {
+        $(selectors.entityTag.dropDownUsersMenu).click();
+        $(selectors.entityTag.user).scrollIntoView();
+        $(selectors.entityTag.user).click;
+    });
```
Comment (Burdishka) [inline test/specs/entityTagNewRecord.js:49]: Here you don't have any verification. I suggest combining all these statements in a single 'it' with all verification that you need to do

File: test/specs/entityTagNewRecord.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        for (let i = 0; i < $(selectors.entityTag.listOfTable).length; i++) {
+            expect($(selectors.entityTag.listOfTable)[i].getText())
+                .to.equal(expected.entityTag.expectedValue[i]);
+        }
+    });
+    it('11/LP-Should verify check-square sign', () => {
+        expect($(selectors.entityTag.checkSqrSign).isExisting())
+            .to.equal(true);
+    });
+});
```
Comment (Burdishka) [inline test/specs/entityTagNewRecord.js:68]: 69 need ad empty line

**Review by Burdishka — APPROVED** [review]

File: test/specs/entityTagNewRecord.js (hunk 2 above)
Comment (LMalushkina) [reply test/specs/entityTagNewRecord.js]: Done

File: test/specs/entityTagNewRecord.js (hunk 1 above)
Comment (LMalushkina) [reply test/specs/entityTagNewRecord.js]: Done

**Pull request merged**